README.md

```diff
@@ -1 +1,2 @@
-# Enumerable_methods
\ No newline at end of file
+# Enumerable_methods
+Project 2 micoverse's ruby course
```

Comment:
You do not have to mention Microverse Ruby course project, you can remove this line